src/tools/run-make-support/src/rustc.rs

```diff
@@ -150,6 +150,20 @@ impl Rustc {
         self
     }
 
+    /// Pass a codegen option.
+    pub fn codegen_option(&mut self, option: &str) -> &mut Self {
+        self.cmd.arg("-C");
+        self.cmd.arg(option);
+        self
+    }
+
+    /// Add a directory to the library search path.
+    pub fn library_search_path<P: AsRef<Path>>(&mut self, path: P) -> &mut Self {
+        self.cmd.arg("-L");
+        self.cmd.arg(path.as_ref());
+        self
+    }
```

Comment:
I would even just call this `L()`

Reply:
That doesn't make it much better than the `-L` in in the bash version, which can be quite opaque to readers, IMO.

Reply:
I don't feel *too* strongly about this, it's fine to keep it `library_search_path` if that seems more readable.

Reply:
It took me a solid minute of browsing online to find "-L" in the rustc docs. Not hard, but can still be made easier by having the function be named after what it does.

Reply:
Probably mention that `library_search_path` is the `-L` option in the doc comment.